### ISSUE TITLE
🛑🛑🛑!!!!!!!IF YOUR HAND DOSENT MOVE READ THIS!!!!!!!!!!!🛑🛑🛑

### ISSUE THREAD

**Issue by seiyaxzavala** [Another-Axiom/GorillaLocomotion#36]

**Comment by seiyaxzavala** [comment]
you need to download Unity Editor version 2021.3.26 just search up  2021.3.26 download and use that version

**Comment by GBCVR** [comment]
will 2021.3.27 work?

**Comment by Froggy990** [comment]
> will 2021.3.27 work?

yes it will any version above 2021.3.26 will work but i just recommend the most recent update for better and easier navigation

**Comment by StarCinnamon232** [comment]
The older Unity Versions now have security risks, I use Unity 6.2, Will the Gtag Loco. still work?

**Comment by cbrincat77** [comment]
starcinnamon232 no the gorilla loco is broken
